### ISSUE TITLE
Prevent users from editing past days

### ISSUE THREAD

**Issue by Fikemox** [alexberryman/DiscordWordle#11]
The Wordle site doesn't have a way for you to go back in time, so the Discord bot should mirror that functionality.

Prevent updates or submissions for Wordles completed before today.

btw: Great bot, thanks for this community add!

**Comment by alexberryman** [comment]
@Fikemox It seems like there are two ways to interpret your request. 

## 1. Delete the `@Discordle update 211 1/6` command
The first part I would like to clarify is "editing past days". I assume this is referring to the `@Discordle update 211 1/6` command to modify a score that has already been registered with the bot. I created this command in case of typos or accidental entries. So far I haven't seen a typo/mistake happen, but it's nice to have that flexibility. I'd be fine with removing this command.

## 2. Prevent yesterday's score from being entered
The other way I could interpret this request is not allowing non-today games to be registered with the bot. If today's game is `214`, don't allow games from `213,212, etc`. I don't intend to implement this change. Since I create weekly scoreboards it's nice for newcomers to bot on a sever to be able to catch up. Many of the users of the bot have been tweeting results or sharing them in other group chats so it's reasonable that they may have yesterday's score available to enter. 

There is also the issue of timezones that would make this difficult to implement. Since it's tomorrow in Japan (UTC+9) they are already on game `215`, but here in Chicago (UTC-6) we're still on game `214`. Each server/user would have to pick a timezone for their "today's game" to be used when enforcing this rule.

## tl;dr
Are you asking for:
1. Delete the `@Discordle update 211 1/6` command?
2. Prevent yesterday's score from being entered?

I can do option 1, but I likely won't do option 2.

**Comment by Fikemox** [comment]
"Yesterday" might have been too specific of a word to use here. Really I meant preventing people from entering any score that isn't for the current day. If you're not up for implementing that, no worries. Forks exist for a reason :)

**Comment by alexberryman** [comment]
Cool. I'll be interested to see how you implement it. 

If you find a solution that you like and works across multiple servers, feel free to open a PR.
